active/jwt_none_vuln.js

```diff
@@ -0,0 +1,134 @@
+// ECMA Script uses the Oracle Nashorn engine, therefore all standard library comes from Java
+// https://docs.oracle.com/javase/8/docs/technotes/guides/scripting/prog_guide/javascript.html
+var Cookie = Java.type("java.net.HttpCookie")
+var Base64 = Java.type("java.util.Base64")
+var String = Java.type("java.lang.String")
+
+// Exploit information, used for raising alerts
+var RISK = 3
+var CONFIDENCE = 3
+var TITLE = "JWT None Exploit"
+var DESCRIPTION = "The application's JWT implementation allows for the usage of the 'none' algorithm, which bypasses the JWT hash verification"
+var SOLUTION = "Use a secure JTW library, and (if your library supports it) restrict the allowed hash algorithms"
```

Comment:
"JTW" > "JWT"

It would be good info description and solution both had periods.